tests/kernel/install_ltp.pm

```diff
@@ -179,16 +179,18 @@ sub upload_runtest_files {
     my $aiurl = autoinst_url();
 
     my $up_script = qq%
-rfiles=\$(ls --file-type $dir)
-for f in \$rfiles; do
-    echo "Uploading ltp-\$f-$tag"
-    curl --form upload=\@$dir/\$f --form target=assets_public $aiurl/upload_asset/ltp-\$f-$tag
-    echo "ltp-\$f-$tag" >> /tmp/ltp-runtest-files-$tag
-done
-curl --form upload=\@/tmp/ltp-runtest-files-$tag --form target=assets_public $aiurl/upload_asset/ltp-runtest-files-$tag
-curl --form upload=\@/root/openposix-test-list-$tag --form target=assets_public $aiurl/upload_asset/openposix-test-list-$tag
+ldir='/tmp/runtest-files-$tag'
+archive="\$ldir.tar.gz"
+mkdir -p \$ldir
+cd \$ldir
+ls --file-type $dir | sed "s/\\(.*\\)/ltp-\\1-$tag/" > \$ldir/ltp-runtest-files-$tag
+cp -v $dir/* ~/openposix-test-list-$tag \$ldir
+tar czvf \$archive *
+ls -la \$archive
+file \$archive
+echo "curl --form upload=\@\$archive --form target=assets_public $aiurl/upload_asset/\$(basename \$archive)"
+curl --form upload=\@\$archive --form target=assets_public $aiurl/upload_asset/\$(basename \$archive)
```

Comment:
Can be used `upload_asset` from `testapi`  ?  Instead of own upload.

Reply:
I can try that, but 1) Richie had probably a reason, why he did it (it was used before).

Reply:
Probably just because it puts the whole process in one script.